backend/src/wamp.js

```diff
@@ -166,6 +166,10 @@ wampHandlers["get-account-details"] = async ([accountId]) => {
       .callViewMethod(lockupAccountId, "get_locked_amount", {})
       .then((balance) => new BN(balance))
       .catch(ignore_if_does_not_exist),
+    nearRpc
+      .callViewMethod(lockupAccountId, "get_reward_fee_fraction", {})
+      .then((fee) => fee)
+      .catch(ignore_if_does_not_exist),
```

Comment:
@frol, here suppose I can to fetch "fee" but I'm not sure. May be not here but in the same way.